include/bbp/sonata/reports.h

```diff
@@ -52,6 +52,18 @@ int sonata_add_element(const char* report_name,
                        uint64_t node_id,
                        uint32_t element_id,
                        double* element_value);
+
+/**
+ * \brief Add an element value to an existing node on a report
+ * \return 0 if operator succeeded, -2 if the report doesn't exist, -3 if the specified node
```

Comment:
Initially it was done this way since it needed a C API for neuron. Now that mod files support C++, this whole API can be changed to a more C++ way. This can be tackled in another PR.